test/integration/node-specific/async_dispose.test.ts

```diff
@@ -1,7 +1,7 @@
 import { expect } from 'chai';
```

Comment:
Now that we run only Node20+, which supports Symbol.asyncDispose natively, we can run this as a part our integration tests.